base/ca/src/main/java/com/netscape/ca/CertificateAuthority.java

```diff
@@ -1836,9 +1843,12 @@ private SingleResponse processRequest(Request req) {
             } else {
                 certStatus = new UnknownInfo();
             }
-        } catch (Exception e) {
```

Comment:
Are we sure there are no other type of exceptions?  This catch all exception in general is a safety net in a pinch, if not the best practice.